flytepropeller/pkg/apis/flyteworkflow/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -548,6 +548,10 @@ func (in *NodeStatus) DeepCopyInto(out *NodeStatus) {
 		*out = new(DynamicNodeStatus)
 		(*in).DeepCopyInto(*out)
 	}
+	if in.ArrayNodeStatus != nil {
+		in, out := &in.ArrayNodeStatus, &out.ArrayNodeStatus
+		*out = (*in).DeepCopy()
+	}
```

Comment:
can you document how you generated the change in this file?

Reply:
I did this manually on our fork when working on an array node feature a couple months ago. I couldn't get the script to run.